politeiad/backendv2/tstorebe/plugins/pi/hooks.go

```diff
@@ -404,14 +498,134 @@ func (p *piPlugin) commentWritesAllowed(token []byte) error {
 		ticketvote.VoteStatusStarted:
 		// Comment writes are allowed on these vote statuses
 		return nil
+	case ticketvote.VoteStatusApproved:
+		// Get billing status to determine whether to allow author updates
```

Comment:
The more nested code is the harder it is to read and reason about.  Can we abstract this code into a function in order to improve readability?